Add feedback bar

hbillings opened the pull request:
<img width="1246" alt="screen shot 2017-11-07 at 5 38 47 pm" src="https://user-images.githubusercontent.com/509309/32525615-83105cf8-c3e2-11e7-8e11-0c40029effdf.png">

<img width="807" alt="screen shot 2017-11-07 at 5 50 16 pm" src="https://user-images.githubusercontent.com/509309/32525958-3293ddca-c3e4-11e7-907b-3eae6ca12087.png">


Todo:
- [x] finish fixing mobile nav
- [ ] ~make it collapsible~
- [x] only apply it to the right pages?

Commit:
move nav around and add static beta bar

Commit:
basic mobile

adborden commented on www.foia.gov/_sass/modules/_feedback_bar.scss:
```diff
@@ -0,0 +1,19 @@
+  }
+  a {
+    border-bottom: 1px solid $color-primary-darkest;
+  }
```
We should just have a few reverse-panel subclasses to add these styles.

adborden committed:
Finishing touches on the beta bar

- Hide the secondary nav at 900px breakpoint to avoid it crashing into the
primary nav
- Show the secondary nav in the mobile menu
- Fix the height on the nav bar on mobile

adborden committed:
Show feedback bar on pages

Avoid showing feedback bar on old content pages, show it on all the others.

adborden committed:
Merge branch 'develop' into adb-beta-bar

adborden committed:
Merge fix

Fix the text color on selected nav items.
